site/weavedns/managing-entries-weavedns.md

```diff
@@ -4,17 +4,13 @@ menu_order: 40
 search_type: Documentation
 ---
 
-
 The following topics are discussed: 
 
 * [Adding and removing extra DNS entries](#add-remove)
 * [Resolving WeaveDNS entries from the Host](#resolve-weavedns-entries-from-host)
 * [Hot-swapping Service Containers](#hot-swapping)
-* [Retaining DNS Entries When Containers Stop](#retain-stopped)
```

Comment:
I looked through the history and I don't see that topic.  I will give it a more thorough look later.